drivers/adc/adc_npcx.c

```diff
@@ -0,0 +1,374 @@
+/*
+ * Copyright (c) 2020 Nuvoton Technology Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT nuvoton_npcx_adc
+
+#include <assert.h>
+#include <drivers/adc.h>
+#include <drivers/clock_control.h>
+#include <kernel.h>
+#include <soc.h>
+
+#define ADC_CONTEXT_USES_KERNEL_TIMER
+#include "adc_context.h"
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(adc_npcx, CONFIG_ADC_LOG_LEVEL);
+
+/* ADC speed/delay values during initialization */
+#define ADC_REGULAR_DLY_VAL     0x03
```

Comment:
Should use tab instead of multiple spaces?

Reply:
Done.